packages/cache-buster/test/cacheBuster.test.ts

```diff
@@ -1,7 +1,7 @@
-import { AssetPack, path } from '@play-co/assetpack-core';
-import { existsSync, readFileSync } from 'fs-extra';
+import { Asset, AssetPack, path } from '@play-co/assetpack-core';
+import { existsSync } from 'fs-extra';
```

Comment:
we cant import like this because of the commonjs and es module compatibility

use `import fs from `'fs-extra'`